cis/gcp/compute/gcp-cis-4.2-compute-block-project-wide-ssh-keys-enabled-for-vm-instances/testdata/mock-tfplan-failure-null-metadata.sentinel

```diff
@@ -0,0 +1,217 @@
+resource_changes = {
```

Comment:
This Sentinel configuration file should not be required if you implement my suggestion.